setup.py

```diff
@@ -70,15 +76,16 @@ def read_file(*path):
     zip_safe=False,  # otherwise ZConfig can't see component.xml
     install_requires=[
         'perfmetrics',
-        'ZODB3>=3.7.0',
+        'ZODB3>=3.7.0,<3.11.0',
```

Comment:
I have a very strong preference to avoid imposing version pins that aren't technically necessary; that cramps everyone's style. If it becomes a massive support burden we can reconsider.